packages/fx-core/src/common/spec-parser/utils.ts

```diff
@@ -340,3 +340,12 @@ export function validateServer(spec: OpenAPIV3.Document): ErrorResult[] {
   }
   return errors;
 }
+
+export function isWellknownResultPropertyName(name: string): boolean {
+  for (let i = 0; i < ConstantString.WellknownResultNames.length; i++) {
```

Comment:
we might can do ConstantString.WellknownResultNames.includes(name.tolowercase()) directly?

Reply:
If name's substring contains wellknown name, I also want to match it. For example, if a name is "xxxItems", I also want to match it.

Reply:
Make sense.